### DISCUSSION TITLE
issue while signing up for student developer pack

### DISCUSSION THREAD

**Discussion by sushi32** [community/community#46083] [GitHub Education]
I am a master's student at University at Buffalo. I am trying to sign up for the student developer pack and no matter what I try or which document I use to verify, the system is giving me some error and I am unable to signup. 
Someone, please help me regarding this.

**Comment by jge162** [comment]
Here you are @sushi32 you can use this link to submit a ticket for help [GitHub Education support](https://support.github.com/contact/education)

**Reply by chakesezechi** [reply]
Could you try enlarging the text of the document you are trying to use? For example, if it's a document with the school letterhead and school ID, I had to use landscape instead of portrait to expand the text.
